pkg/tsdbctl/tsdbctl.go

```diff
@@ -161,10 +161,10 @@ func (rc *RootCommandeer) populateConfig(cfg *config.V3ioConfig) error {
 		if slash == -1 || len(rc.v3ioPath) <= slash+1 {
 			if rc.container != "" {
 				cfg.Container = rc.container
-				cfg.V3ioUrl = rc.v3ioPath
-			} else {
+			} else if cfg.Container == "" {
```

Comment:
something is off here, cfg.Container will be empty for sure as it's not assigned anywhere before

Reply:
look at the signature `func (rc *RootCommandeer) populateConfig(cfg *config.V3ioConfig) error {`
`cfg` is input to this methos and it may come with all the settings loaded from some source